res/values-de/strings.xml

```diff
@@ -117,7 +117,7 @@
     <string name="Private">"Privat"</string>
     <string name="Public">"Öffentlich"</string>
     <string name="Bot">"Bot"</string>
-    <string name="You">"Sie"</string>
+    <string name="You">"Du"</string>
```

Comment:
Please return here as "Sie", because it's used at formal contexts.

Reply:
I added it back to Sie

Reply:
Thanks, please mark this as resolved by tapping the "Resolve" button